docs/tutorial/client-side-encryption.txt

```diff
@@ -55,7 +55,9 @@ more than one encryption key or create them dynamically.
    // To store the key ID for later use, you can use serialize or var_export
    $keyId = $clientEncryption->createDataKey('local', ['keyAltNames' => ['my-encryption-key']]);
 
-.. seealso:: :manual:`Encryption Key Management </csfle/fundamentals/manage-keys/>` in the MongoDB manual
+.. seealso::
+
+   :manual:`Encryption Key Management </csfle/fundamentals/manage-keys/>` in the MongoDB manual
```

Comment:
Issue:

This URL doesn't current exist. 

Suggestion:

I think this is the URL you intended:
```suggestion
   :manual:`Encryption Key Management </core/csfle/fundamentals/manage-keys/>` in the MongoDB manual
```

Here's the `manual` role definition:

https://github.com/mongodb/snooty-parser/blob/latest/snooty/rstspec.toml#L1346-L1348

The `%s` represents the substitution for the content provided in the angled brackets.